wallet/src/main/java/com/android/identity_credential/wallet/presentation/OpenID4VPPresentationActivity.kt

```diff
@@ -522,7 +522,9 @@ class OpenID4VPPresentationActivity : FragmentActivity() {
 
         val vpTokenByteArray = generateVpToken(consentFields, credentialToUse, trustPoint, authorizationRequest, sessionTranscript)
         Logger.i(TAG, "Setting vp_token: ${vpTokenByteArray.decodeToString()}")
-        val vpToken = Base64.UrlSafe.encode(vpTokenByteArray)
+        val vpToken = if (credentialToUse is MdocCredential) {
+            Base64.UrlSafe.encode(vpTokenByteArray).replace("=", "")
```

Comment:
I think this would provide the same result, but makes the intent a little clearer:
Base64.UrlSafe.withPadding(Base64.PaddingOption.ABSENT_OPTIONAL).encode(vpTokenByteArray)

Reply:
discussed offline; found this would require updating kotlin from 2.0.0 to 2.0.20 (as well as ksp to 2.0.20-1.0.25) in `gradle/libs.versions.toml` which also necessitates changes in identity-sdjwt where Base64 is used; decided to push these changes to later